block-canvas/style.css

```diff
@@ -39,37 +38,14 @@ body {
 /*
  * Control the hover stylings of outline block style.
  * Unnecessary once block styles are configurable via theme.json
- * https://github.com/WordPress/gutenberg/issues/42794
+ * https://github.com/WordPress/gutenberg/issues/42794 and https://github.com/WordPress/gutenberg/issues/44417
  */
 .wp-block-button.is-style-outline>.wp-block-button__link:not(.has-background):hover {
 	background-color: var(--wp--preset--color--secondary);
 	color: var(--wp--preset--color--background);
 	border-color: var(--wp--preset--color--secondary);
 }
 
-/**
- * Currently table styles are only available with 'wp-block-styles' 
- * theme support (block css) thus the following needs to be included
- * since 'wp-block-styles' aren't used for this theme.
- * https://github.com/WordPress/gutenberg/issues/45065
- */
-.wp-block-table thead {
-	border-bottom: 3px solid;
-}
-.wp-block-table tfoot {
-	border-top: 3px solid;
```

Comment:
I think these are still needed too.

Reply:
We can remove these lines now as Core includes them in the default styles: https://github.com/WordPress/gutenberg/pull/46567. 

We should probably wait for GB 14.9 to be released before merging this.